### PR TITLE
#167916714 Update codebase to ES6

### PR DESCRIPTION
**Pull request by mthamayor**
#### What does this PR do?
Update codebase from ES5 to ES6

#### Description of Task to be completed?
- Refactor codebase to use ES6 features

#### How should this be manually tested?
None

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167916714

#### Screenshots (if appropriate)
None

#### Questions:
None

### CODE REVIEW

**Comment by okafor-chidimma** [comment]
Reviewed...Approve for merge

**Review by okafor-chidimma — APPROVED** [review]

**Review by okabamac — APPROVED** [review]
Looks good to me

File: src/config/index.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-module.exports = {
-    secret:
-        process.env.NODE_ENV === "production" ? process.env.SECRET : "secret"
+const index = {
+  secret:
+        process.env.NODE_ENV === 'production' ? process.env.SECRET : 'secret'
 };
+
+export default index;
```
[the remark this answers is not in the document]
Comment (davidfrank96) [reply src/config/index.js:6]: Nice work ...Approved

**Review by davidfrank96 — APPROVED** [review]
nice work...Approved, keep it up

**Review by chibytez — APPROVED** [review]

**Review by Stylll — APPROVED** [review]

**Review by chibytez — APPROVED** [review]

**Pull request merged**